lib/httplib/csrf/csrf.go

```diff
@@ -121,3 +128,20 @@ func save(encodedToken string, w http.ResponseWriter) string {
 	w.Header().Add("Vary", "Cookie")
 	return encodedToken
 }
+
+// TODO(kimlisa): DELETE IN v19.0 (csrf)
+// Deprecated: do not use (only here to support backwards compat for old plugin endpoints)
+// VerifyFormField checks if HTTP form value matches the cookie.
+func VerifyFormField(r *http.Request) error {
```

Comment:
Does this need to be exported? Can we instead inline this logic (with a `DELETE IN` comment) directly in `WithCSRFProtection`?

The less of these things we make publicly available, the less chance that someone starts using them before we're able to remove completely in v19.